repository.rdf

```diff
@@ -417,16 +336,6 @@
 
 
 
-    <!-- http://fedora.info/definitions/v4/repository#Configuration -->
-
-    <owl:Class rdf:about="http://fedora.info/definitions/v4/repository#Configuration">
```

Comment:
I don't think this would break transform (but I haven't tried), but it does beg a question. We version the ontology but always use the alias to the current version. It seems like we would need to fix the ontology version before we released Fedora. Just my two cents.